src/plugin/PluginBase.php

```diff
@@ -174,7 +175,7 @@ private function registerYamlCommands() : void{
 
 			$newCmd->setAliases($aliasList);
 
-			$newCmd->setPermission($data->getPermission());
+			$newCmd->setPermission(PermissionManager::getInstance()->getPermission($data->getPermission()));
```

Comment:
This introduces a security hole, `PermissionManager::getPermission()` may return `null`

Reply:
No because this permission is already registered at https://github.com/pmmp/PocketMine-MP/blob/stable/src/plugin/PluginManager.php#L195

Reply:
The permission may not have been declared.

Reply:
so i need to create new Permission here ?

Reply:
No, it needs to check whether the permission exists before proceeding. Previously `setPermission()` would have checked it.